templates/activity/parts/activity-effects.hbs

```diff
@@ -1,29 +1,38 @@
 <fieldset>
-    <legend>{{ localize "DND5E.ACTIVITY.FIELDS.effects.label" }}</legend>
-    <multi-select name="appliedEffects">
-        {{ selectOptions allEffects }}
-    </multi-select>
-    <ul class="separated-list effects-list">
+    <legend>
+        <span>{{ localize "DND5E.ACTIVITY.FIELDS.effects.label" }}</span>
+        <button type="button" class="unbutton control-button" data-action="addEffect"
+                data-tooltip="DND5E.EFFECT.Action.Create"
+                aria-label="{{ localize "DND5E.EFFECT.Action.Create" }}">
+            <i class="fas fa-plus" inert></i>
+        </button>
+    </legend>
+    <div class="form-group">
+        <label>{{ localize "DND5E.EFFECT.Label" }}</label>
+        <div class="form-fields">
+            <multi-select name="appliedEffects" class="hidden-tags">
+                {{ selectOptions allEffects }}
+            </multi-select>
+        </div>
+    </div>
+    <ul class="separated-list effects-list unlist">
         {{#each appliedEffects}}
         <li data-index="{{ @index }}" data-effect-id="{{ effect.id }}" data-expand-id="effects.{{ effect.id }}">
             {{> ".effect" }}
         </li>
         {{/each}}
     </ul>
-    <button type="button" class="unbutton" data-action="addEffect">
-        {{ localize "DND5E.EFFECT.Action.Create" }}
-    </button>
 </fieldset>
 
 {{#*inline ".effect"}}
 <div class="details flexrow">
     <img class="gold-icon" src="{{ effect.img }}" alt="{{ effect.name }}">
     {{{ dnd5e-linkForUuid effect.uuid }}}
```

Comment:
This causes a bug when viewing an activity within a compendium because it uses `fromUuidSync` which cannot retrieve embedded documents in that context.